Backpack/src/main/res/values/backpack.dimensions.spacing.xml

```diff
@@ -17,17 +17,11 @@
   limitations under the License.
 -->
 <resources>
-
-	<dimen name="bpkSpacingSm">4dp</dimen>
-
-	<dimen name="bpkSpacingMd">8dp</dimen>
-
-	<dimen name="bpkSpacingBase">16dp</dimen>
-
-	<dimen name="bpkSpacingLg">24dp</dimen>
-
-	<dimen name="bpkSpacingXl">32dp</dimen>
-
-	<dimen name="bpkSpacingXxl">40dp</dimen>
-
- </resources>
+  <dimen name="bpkSpacingSm">4dp</dimen>
+  <dimen name="bpkSpacingMd">8dp</dimen>
+  <dimen name="bpkSpacingBase">16dp</dimen>
+  <dimen name="bpkSpacingLg">24dp</dimen>
+  <dimen name="bpkSpacingXl">32dp</dimen>
+  <dimen name="bpkSpacingXxl">40dp</dimen>
+  <dimen name="bpkSpacingNone">0dp</dimen>
```

Comment:
none was filtered out before for some reason, but that would require more reworking of the current system - I don't see harm in having it